.github/workflows/manual-test-matrix-workflow.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
             "*jmx*",
             "*jpa*",
             "*jsf*",
+            "*mail*",
```

Comment:
could you please rebase the branch? The regexes has been changed in https://github.com/hal/berg/pull/18/files#diff-88d5f3bd580fcbf6d0e9294a1ee85d5dd61c153d30f4b8f0d1754d188aa84d88

Reply:
rebase done.

Reply:
nope, we are now placing tests into the submodules listed. Before we used to run `--specs=cypress/e2e/*${MODULE_NAME}*` now we are running `--specs=cypress/e2e/${MODULE_NAME}/*cy.ts`. So just get rid of the asterisks and it will be okay 🙂